frontend/src/components/associations/marketplace/counter/CounterOrderMaker.tsx

```diff
@@ -103,38 +103,61 @@ export const CounterOrderMaker = ({
             });
         }
 
-        queryCache.refetchQueries("marketplace.transactions.list");
+        await queryCache.refetchQueries("marketplace.transactions.list");
     };
 
+    const basketIsEmpty = Object.keys(basket).length === 0;
+
     return (
         <>
-            <Card>
-                <Card.Header>
-                    Solde :{" "}
-                    {balanceStatus === "loading" ? (
-                        <em>Chargement</em>
-                    ) : balanceStatus === "error" ? (
-                        <em>Erreur {balanceError} : balance</em>
-                    ) : (
-                        `${(balance as { balance: number }).balance}€`
-                    )}
-                </Card.Header>
-            </Card>
-
-            <RefundForm customer={customer} marketplaceId={marketplaceId} />
-
-            <ProductSearch
-                basket={basket}
-                addToBasket={addToBasket}
+            <RefundForm
+                customer={customer}
                 marketplaceId={marketplaceId}
+                className={"float-right mt-2"}
             />
 
-            {/* Show order summary */}
-            <OrderSummary
-                basket={basket}
-                removeFromBasket={removeFromBasket}
-                makeOrder={makeOrder}
-            />
+            <div className="d-flex justify-content-center mt-4">
+                <Col md={"6"}>
+                    <Card>
+                        <Card.Body className="text-center">
+                            <h5>{customer.id}</h5>
+                            <div className="display-2 font-weight-bold">
+                                {balanceStatus === "loading" ? (
+                                    <em>Chargement</em>
```

Comment:
This is blown up because displayed in size `display-2` when loading
It would be better to have a Spinner, or something else more discreet. Nothing is good too.